deploy/crontablist.sh

```diff
@@ -60,14 +60,27 @@ echo -n "0 0 * * * "
 echo -n "php ${SERVICE_DIR}/artisan ops:license:fetch"
 echo ""
 
+_CREDENTIALS="--user=${VENDOR_USER} --password=${VENDOR_USER}"
+
 _DB="${VENDOR_NAME}_${SERVICE_NAME}"
-_TABLE="network_event_logs"
-_CONDITION="created_at < CURRENT_DATE - INTERVAL 32 DAY"
+
+__DATE=$(mysql ${_CREDENTIALS} -e "SELECT CURRENT_DATE - INTERVAL 32 DAY" --batch | tail -1)
+_CONDITION="created_at < '${__DATE}'"
+
 _FILE_DIR="${VENDOR_DIR}/.backup"
+
+_TABLE="network_event_logs"
 _FILE="${_FILE_DIR}/${_TABLE}-\$(date -u -Iseconds).sql"
-_CREDENTIALS="--user=${VENDOR_USER} --password=${VENDOR_USER}"
 
-echo -n "30 0 * * * "
+echo -n "0 1 * * * "
+echo -n "mysqldump ${_CREDENTIALS} --no-tablespaces --no-create-db --no-create-info --where=\"${_CONDITION}\" --result-file=${_FILE} ${_DB} ${_TABLE}"
+echo -n " && "
+echo -n "mysql ${_CREDENTIALS} --execute=\"DELETE FROM ${_TABLE} WHERE ${_CONDITION}\"" ${_DB}
```

Comment:
I'd move `mysqldump` and `mysql` commands to the dedicated (new) bash script (e.g.  `backup.sh`).